basics/signals/package.json

```
@@ -45,13 +45,16 @@
   },
   "dependencies": {
     "@jupyterlab/application": "^3.0.0-rc.15",
+    "@jupyterlab/apputils": "^3.0.3",
     "@jupyterlab/launcher": "^3.0.0-rc.15",
     "@jupyterlab/mainmenu": "^3.0.0-rc.15",
     "@jupyterlab/translation": "^3.0.0-rc.15",
     "@lumino/algorithm": "^1.3.3",
     "@lumino/coreutils": "^1.5.3",
     "@lumino/datagrid": "^0.3.1",
-    "@lumino/disposable": "^1.4.3"
+    "@lumino/disposable": "^1.4.3",
+    "@lumino/signaling": "^1.3.3",
+    "@lumino/widgets": "^1.3.3"
```

```suggestion
    "@lumino/widgets": "^1.16.1"
```

Alignment with jupyterlab dependency constrain

---

Fixed as requested.  Thanks for the guidance.